portal/mailservice/mailer.py

```diff
@@ -0,0 +1,35 @@
+import smtplib
+
+from config import HOST, PORT, TLS, USER, PASSWORD, FROM, SEND_EMAILS
+
+
+class Mailer:
+    def __init__(self, send_emails=SEND_EMAILS):
+        self.__send_emails = send_emails
```

Comment:
Yeah, I realized it in the end, left as a comment because this may suffice a comment in the code itself.